src/software/jetson_nano/ansible/playbooks/setup_nano.yml

```diff
@@ -141,6 +141,17 @@
           args:
             chdir: ~/
 
+        - name: Load spidev on boot
```

Comment:
This seems sketchy if setup_nano is run multiple times. From quick search unknown if duplicates mess things up

Reply:
fixed this because I added it to the kernel iso so we don't need to add it ourselves